src/plugins/mer/meremulatordevice.cpp

```diff
@@ -692,6 +765,23 @@ bool MerEmulatorDeviceManager::restorePorts(const MerEmulatorDevice::Ptr &device
     return true;
 }
 
+bool MerEmulatorDeviceManager::restoreSystemSettings(const MerEmulatorDevice::Ptr &device)
```

Comment:
Unless I missed something, this should be merged with restorePorts(), using restoreSystemSettings as the new name. All comments should be updated accordingly (there is also a comment where the function is called)